cli/src/pcluster/cli_commands/commands.py

```diff
@@ -61,13 +61,9 @@ def _parse_config_file(config_file, fail_on_config_file_absence=True):
             LOGGER.debug("Specified configuration file %s doesn't exist.", config_file)
     else:
         LOGGER.debug("Parsing configuration file %s", config_file)
-    try:
-        return load_yaml_dict(file_path=config_file)
-    except Exception as e:
-        utils.error(
-            "Error parsing configuration file {0}.\nDouble check it's a valid Yaml file. "
-            "Error: {1}".format(config_file, str(e))
-        )
+    with open(config_file) as conf_file:
```

Comment:
1. debug message is wrong.
2. We should keep try/except block to catch I/O errors.